framework/src/modules/token/token_module.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright © 2020 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+import { BaseModule } from '../base_module';
+
+interface AccountBalance {
+	balance: bigint;
+}
+
+export class TokenModule extends BaseModule<AccountBalance> {
+	public name = 'token';
+	public type = 2;
+	public accountSchema = {
+		type: 'object',
+		properties: {
+			balance: {
+				dataType: 'uint64',
+			},
+		},
+		default: {
+			balance: BigInt(0),
+		},
```

Comment:
We need to decide on this as a pattern, if we support `default` as part of the schema then we have to update the configuration here. 

https://github.com/LiskHQ/lisk-sdk/blob/213506f490002f45eec5caef143ef764f7ddeca0/elements/lisk-validator/src/lisk_validator.ts#L33

Reply:
it is not for the validation, but chain needs to know all the default value to create the default account by combining all the schema.

Reply:
I do understand our use case, but if we have `default` key inside a schema, developers may get it confused with validator default values.

Reply:
The config has the same usage of default, and to keep it consistent within framework we will go with this approach